python/tvm/relay/op/strategy/arm_cpu.py

```diff
@@ -235,6 +235,28 @@ def conv2d_strategy_arm_cpu(attrs, inputs, out_type, target):
                     wrap_topi_schedule(topi.arm_cpu.schedule_depthwise_conv2d_nhwc),
                     name="depthwise_conv2d_nhwc.arm_cpu",
                 )
+
+            # Optimized special case depthwiseConv2D operation. Requires a 3x3 kernel, a
+            # NHWC layout, a HWOI kernel layout (which we would ideally rearrange), no dilation,
+            # "SAME" padding, int8 inputs and outputs, the same number of input and output
```

Comment:
I noticed that `depthwise_conv2d_nhwc_dsp_compute()` supports VALID padding as well. Is it just the comment that requires editing or there are other bits that limit the support to same spatial dimensions for IO?

Reply:
Fixed! The schedule supports any form of padding - SAME, VALID, or a custom tuple.